### PR TITLE
docs: add bi-weekly report (1204-1217)

### PR DESCRIPTION
**Pull request by saltism**

### CODE REVIEW

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+- [新增 `jwe decrypt` 插件](https://github.com/apache/apisix/pull/10252)（贡献者：[fishioon](https://github.com/fishioon))
+
+- [`openid-connect` 插件新增更多属性](https://github.com/apache/apisix/pull/10591)（贡献者：[kayx23](https://github.com/kayx23))
+
+- [`CORS` 插件支持 Timing-Allow-Origin](https://github.com/apache/apisix/pull/9365)（贡献者：[skimdz86](https://github.com/skimdz86))
+
+- [新增 `brotli` 插件](https://github.com/apache/apisix/pull/10515)（贡献者：[yuweizzz](https://github.com/yuweizzz))
+ 
+- [使用 lua-resty-events 作为默认事件机制](https://github.com/apache/apisix/pull/10550)（贡献者：[bzp2010](https://github.com/bzp2010))
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
- [使用新的 lua-resty-events 作为默认事件机制](https://github.com/apache/apisix/pull/10550)（贡献者：[bzp2010](https://github.com/bzp2010))
```

File: blog/en/blog/2023/12/20/bi-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+- [Add `jwe decrypt` plugin](https://github.com/apache/apisix/pull/10252) (Contributor: [fishioon](https://github.com/fishioon))
+
+- [Add more attributes to `openid-connect` plugin](https://github.com/apache/apisix/pull/10591) (Contributor: [kayx23](https://github.com/kayx23))
+
+- [`CORS` plugin supports Timing-Allow-Origin](https://github.com/apache/apisix/pull/9365) (Contributor: [skimdz86](https://github.com/skimdz86))
+
+- [Add `brotli` plugin](https://github.com/apache/apisix/pull/10515) (Contributor: [yuweizzz](https://github.com/yuweizzz))
+
+- [Use lua-resty-events as the default event mechanism](https://github.com/apache/apisix/pull/10550) (Contributor: [bzp2010](https://github.com/bzp2010))
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
- [Add new lua-resty-events as the default event mechanism](https://github.com/apache/apisix/pull/10550) (Contributor: [bzp2010](https://github.com/bzp2010))
```

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+从 2023.12.04 至 2023.12.17，有 21 名开发者提交了 46 个 commit，为 Apache APISIX 做出了重要贡献。感谢这些伙伴们对 Apache APISIX 的无私支持！正是因为你们的付出，才能让 Apache APISIX 项目不断改进、提升和壮大。
+
+最近，我们对一些功能进行了新增和改进，更新内容总结如下：
+
+1. 新增 `jwe decrypt` 插件
+
+2. `openid-connect` 插件新增更多属性
+
+3. `CORS` 插件支持 Timing-Allow-Origin
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
3. `CORS` 插件支持 Timing-Allow-Origin 响应头
```

File: blog/en/blog/2023/12/20/bi-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+From 12.04 to 12.17, a total of 21 contributors made 46 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+We have recently added and enhanced several plugins, and here is a summary of the updates:
+
+1. Add `jwe decrypt` plugin
+
+2. Add more attributes to `openid-connect` plugin
+
+3. `CORS` plugin supports Timing-Allow-Origin
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
3. Add support for the Timing-Allow-Origin header in `CORS` plugin
```

File: blog/en/blog/2023/12/20/bi-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/12/20/GycOBJie_NEW.png)
+
+## Highlight of Recent Feature
+
+- [Add `jwe decrypt` plugin](https://github.com/apache/apisix/pull/10252) (Contributor: [fishioon](https://github.com/fishioon))
+
+- [Add more attributes to `openid-connect` plugin](https://github.com/apache/apisix/pull/10591) (Contributor: [kayx23](https://github.com/kayx23))
+
+- [`CORS` plugin supports Timing-Allow-Origin](https://github.com/apache/apisix/pull/9365) (Contributor: [skimdz86](https://github.com/skimdz86))
```
Comment (Yilialinn) [inline blog/en/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
- [Add support for the Timing-Allow-Origin header in `CORS` plugin](https://github.com/apache/apisix/pull/9365) (Contributor: [skimdz86](https://github.com/skimdz86))
```

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- [新增 `jwe decrypt` 插件](https://github.com/apache/apisix/pull/10252)（贡献者：[fishioon](https://github.com/fishioon))
+
+- [`openid-connect` 插件新增更多属性](https://github.com/apache/apisix/pull/10591)（贡献者：[kayx23](https://github.com/kayx23))
+
+- [`CORS` 插件支持 Timing-Allow-Origin](https://github.com/apache/apisix/pull/9365)（贡献者：[skimdz86](https://github.com/skimdz86))
+
+- [新增 `brotli` 插件](https://github.com/apache/apisix/pull/10515)（贡献者：[yuweizzz](https://github.com/yuweizzz))
+ 
+- [使用 lua-resty-events 作为默认事件机制](https://github.com/apache/apisix/pull/10550)（贡献者：[bzp2010](https://github.com/bzp2010))
+
```
Comment (yzeng25) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]: Fix lint, delete extra space in line 46.

```suggestion
- [新增 `brotli` 插件](https://github.com/apache/apisix/pull/10515)（贡献者：[yuweizzz](https://github.com/yuweizzz))

- [使用 lua-resty-events 作为默认事件机制](https://github.com/apache/apisix/pull/10550)（贡献者：[bzp2010](https://github.com/bzp2010))

```

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+1. 新增 `jwe decrypt` 插件
+
+2. `openid-connect` 插件新增更多属性
+
+3. `CORS` 插件支持 Timing-Allow-Origin
+
+4. 新增 `brotli` 插件
+
+5. 使用 lua-resty-events 作为默认事件机制
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
5. 使用新的 lua-resty-events 作为默认事件机制
```

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/12/20/GycOBJie_NEW.png)
+
+## 近期亮点功能
+
+- [新增 `jwe decrypt` 插件](https://github.com/apache/apisix/pull/10252)（贡献者：[fishioon](https://github.com/fishioon))
+
+- [`openid-connect` 插件新增更多属性](https://github.com/apache/apisix/pull/10591)（贡献者：[kayx23](https://github.com/kayx23))
+
+- [`CORS` 插件支持 Timing-Allow-Origin](https://github.com/apache/apisix/pull/9365)（贡献者：[skimdz86](https://github.com/skimdz86))
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
- [`CORS` 插件支持 Timing-Allow-Origin 响应头](https://github.com/apache/apisix/pull/9365)（贡献者：[skimdz86](https://github.com/skimdz86))
```

File: blog/zh/blog/2023/12/20/bi-weekly-report.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
+---
+title: "社区双周报 (12.04 - 12.17)"
+keywords: ["Apache APISIX", "API 网关", "社区周报", "贡献者"]
+description: Apache APISIX 社区的双周报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/12/20/hTVExnq0_ZH.png
+---
+> 最近，我们新增并改进了 Apache APISIX 的部分功能，如新增 `multi-auth` 和 `brotli` 插件、`openid-connect` 插件新增更多属性、`CORS` 插件支持 Timing-Allow-Origin、使用新的 lua-resty-events 作为默认事件机制。详情请阅读本期双周报。
```
Comment (Yilialinn) [inline blog/zh/blog/2023/12/20/bi-weekly-report.md]:
```suggestion
> 最近，我们新增并改进了 Apache APISIX 的部分功能，如新增 `multi-auth` 和 `brotli` 插件、`openid-connect` 插件新增更多属性、`CORS` 插件支持 Timing-Allow-Origin 响应头、使用新的 lua-resty-events 作为默认事件机制。详情请阅读本期双周报。
```

**Review by Yilialinn — APPROVED** [review]

**Review by SkyeYoung — DISMISSED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**